api/nakadi-event-bus-api.yaml

```diff
@@ -1217,6 +1221,7 @@ definitions:
       - name
       - category
       - owning_application
+      - enrichment_strategies
```

Comment:
But in general I agree. These categories make me think of:
![](https://s-media-cache-ak0.pinimg.com/236x/ae/20/e2/ae20e26ff6c97c4664cae41e88d1a6e0.jpg)